examples/rehastimp24_example.py

```diff
@@ -0,0 +1,170 @@
+from pyScienceMode2.channel import Channel, Point
+from pyScienceMode2.rehastim_interface import StimulatorP24 as St
+
+"""
+This example shows how to use the RehastimP24 device. 
+There are several commands divided into three levels (general, low and mid).
+You can't call commands from different levels, you must first close the current level
+to be able to use commands from another one.
+"""
+
+# list which contains the channels you want to use
+list_channels = []
+
+# list which contains the points you want to use
+list_stimulation_points = []
+
+# Create an object channel
+channel_1 = Channel(
+    no_channel=1, name="Biceps", amplitude=40.1, pulse_width=500, frequency=35, device_type="RehastimP24"
+)
+channel_2 = Channel(
+    mode="Doublet",
+    no_channel=2,
+    amplitude=40,
+    pulse_width=500,
+    name="Triceps",
+    frequency=35,
+    ramp=5,
+    device_type="RehastimP24",
+)
+channel_3 = Channel(
+    mode="Triplet",
+    no_channel=3,
+    amplitude=40,
+    pulse_width=500,
+    name="Triceps",
+    frequency=15,
+    ramp=15,
+    device_type="RehastimP24",
+)
+
+# Create an object stimulator
+stimulator = St(port="COM4", show_log="Partial")  # Enter the port on which the rehastim is connected
+
+# Add the channels you want to stimulate to the list.
+list_channels.append(channel_1)
+list_channels.append(channel_2)
+# list_channels.append(channel_3)
+
+"""
+General level commands. 
+In this level you can get several information about the device.
+"""
+
+# stimulator.get_battery_status()
+# stimulator.get_stim_status()
+# stimulator.get_main_status()
+# stimulator.get_all()
+# stimulator.reset()
+
+"""
+Low level commands.
+In this level you can configure a custom shape pulse. 
+"""
+
+"""
+Init the ll stimulation. Use it before starting the stimulation or after stopping it.
+"""
+stimulator.ll_init()
+
+# Create a point with the configuration you want to create your own shape pulse.
+point1 = Point(500, 20)
+point2 = Point(500, -20)
+point3 = Point(500, 10)
+point4 = Point(500, -10)
+
+# Add the points you want to use to the list
+list_stimulation_points.append(point1)
+list_stimulation_points.append(point2)
+list_stimulation_points.append(point3)
+list_stimulation_points.append(point4)
+
+"""
+Start the ll stimulation with the list of points provided. 
+It is possible to update the parameters of the point by giving a new list of points.
+If you set the safety flag to False, it will not check if the stimulation points  provided are symmetrical
+for a muscle loading and unloading phase. 
+"""
+stimulator.start_ll_channel_config(no_channel=1, points=list_stimulation_points, stim_sequence=3, pulse_interval=500.5)
```

Comment:
I think the difference between ll and high level is not that clear... LL is like single pulse ?

Reply:
If that so I think it must be more clear and easy to use.

Reply:
In readme ? or here

Reply:
I mean in the code itself, if it is a function to stim only one channel may be call it stim.start_one_channel_stimulation() (also include the init inside this function as you send no parameters into the init). Maybe find another name but more explicit than ll_mode...

Reply:
renamed in stim.start_one_channel_stimulation(). Indeed , This allows for the stimulation of a single channel, but it is capable of stimulating at a higher frequency.